### PR TITLE
Botpage

### PR DESCRIPTION
**Pull request by lefherz**
addressing #201 

I didn't add documentation, only split them up and left a reference at the old place. Not sure if more documentation is necessary.

### CODE REVIEW

**Comment by dkg** [comment]
I wholeheartedly approve of moving the bot info to a separate page from dev-status.

However, this branch seems to include some of the changes from the wordingimprovement branch, which seems a bit distracting compared to the title of the PR ("botpage").  Maybe this would be better as a separate PR, rebased off of the master branch directly?

File: doc/dev-status.rst [hunk 1]
```diff
@@ -91,31 +91,9 @@ Source code:
...
-ssh access to bot account
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You can ssh to the bot account: ``ssh -l bot mail.autocrypt.org``,
-ECDSA key fingerprint is ``SHA256:4RWh81zOd/Pgq3mHhKpyLdVZJfOpq+DgqKheUIhJgWQ``.
-Ask on IRC to get your SSH key added (anyone already with access
-to the bot@autocrypt.org account can add it to ``.ssh/authorized_keys``).
+There is an Autocrypt Bot which accepts and sends mails with Autocrypt
+headers. Just write an E-Mail to bot@autocrypt.org. You can read more
+Details :ref:`here<bot>`.
```
Comment (dkg) [inline doc/dev-status.rst]: [please do not use "here" as link text!](http://synapticloop.com/blog/2008/11/click-here-considered-harmful/)

**Comment by lefherz** [comment]
I thought I dropped those changes. probably not enough of them^^

**Comment by lefherz** [comment]
I opened a better PR: #298

**Pull request closed without merging**